lightning/src/ln/msgs.rs

```diff
@@ -268,6 +288,8 @@ pub struct FundingSigned {
 	pub channel_id: [u8; 32],
 	/// The signature of the channel acceptor (fundee) on the initial commitment transaction
 	pub signature: Signature,
+	/// The partial signature of the channel acceptor (fundee)
+	pub partial_signature_with_nonce: Option<PartialSignatureWithNonce>,
```

Comment:
can be called `fundee_partial_signature_with_nonce` even if more verbose than spec